NitroxServer-Subnautica/Program.cs

```diff
@@ -103,14 +132,12 @@ private static Assembly CurrentDomainOnAssemblyResolve(object sender, ResolveEve
             }
 
             // Read assemblies as bytes as to not lock the file so that Nitrox can patch assemblies while server is running.
-            using (FileStream stream = new FileStream(dllPath, FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
-            using (MemoryStream mstream = new MemoryStream())
-            {
-                stream.CopyTo(mstream);
-                Assembly assembly = Assembly.Load(mstream.ToArray());
-                resolvedAssemblyCache[dllPath] = assembly;
-                return assembly;
-            }
+            using FileStream stream = new FileStream(dllPath, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
+            using MemoryStream mstream = new MemoryStream();
```

Comment:
I must admit that I'm not a big fan of the alternative syntax of the using statement